enterprise-suite/tests/smokecommon

```diff
@@ -108,13 +108,13 @@ test_summary() {
     exit $TEST_FAILURES
 }
 
-# Get URL for NodePort assignment
+# Get URL for accessing console externally
 # nodeport <name>
 nodeport() {
-  if [[ $CURRENT_CONTEXT == "minishift" ]]; then
```

Comment:
Doesn't need to work on minishift anymore or does openshift cover that too?

Reply:
We don't test on minishift, so I have no idea if this code even works. It didn't work out of the box on openshift. We should avoid keeping code that we never run, it just rots.

Reply:
We don't test on minishift from Travis, but I wonder if @srikanthops was using this...

Reply:
Setting the context to "openshift" should have the same effect.